server/models/Profile.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type:String,
+        required:true,
+        maxlength:32,
+        trim:true,   
+    }, 
+    lastName: {
+        type:String,
+        required:true,
+        maxlength:32,
+        trim:true,
+    },
+    description: {
+        type:String,
+    },
+    availability:{
+        type:[String],
+    },
+    profilePic: {
+        type:String,
+    }
```

Comment:
I think just `image` would be an appropriate name. It's already under the profile schema,